src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace ContaoCommunityAlliance\Translator\ContaoManager;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\ManagerBundle\ContaoManagerBundle;
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use ContaoCommunityAlliance\Contao\Bindings\CcaEventsContaoBindingsBundle;
+use ContaoCommunityAlliance\Contao\EventDispatcher\CcaEventDispatcherBundle;
+use ContaoCommunityAlliance\Translator\CcaTranslatorBundle;
+
+/**
+ * Plugin for the Contao Manager.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class Plugin implements BundlePluginInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getBundles(ParserInterface $parser)
+    {
+        return [
+            BundleConfig::create(CcaTranslatorBundle::class)
+                ->setLoadAfter(
+                    [
+                        ContaoCoreBundle::class,
+                        ContaoManagerBundle::class,
+                        CcaEventDispatcherBundle::class,
+                        CcaEventsContaoBindingsBundle::class
```

Comment:
This might work, we have to check however if anyone else ever reacted to this event and therefore will now not get anything anymore.
I know this is unlikely but I wanted to point it out anyway.